shell/common/api/electron_api_clipboard_win.cc

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2020 Microsoft, Inc.
+// Use of this source code is governed by the MIT license that can be
+// found in the LICENSE file.
+
+#include "shell/common/api/electron_api_clipboard.h"
+
+#include <shellapi.h>
+#include <shlobj.h>
+
+#include "base/win/scoped_hglobal.h"
+
+namespace electron {
+
+namespace api {
+
+std::vector<base::FilePath> Clipboard::ReadFilePaths() {
+  std::vector<base::FilePath> result;
+  ::OpenClipboard(NULL);
```

Comment:
Chromium uses `ScopedClipboard` which has more complex logic: https://source.chromium.org/chromium/chromium/src/+/master:ui/base/clipboard/clipboard_win.cc;l=62;drc=510d3cc7ecba433b10d6854f3623ed4ca8fb72d7

Should we replicate it?

Reply:
On Windows the clipboard may be not available when it is being used by other apps, Chromium solves that by retrying after blocking for 5 seconds. I'm not sure if we need to replicate it, but it is easy to if you think it is necessary.